From befeadea9a728b36e6f70a0dcab1210a6141c545 Mon Sep 17 00:00:00 2001
From: serge-sans-paille <serge.guelton@telecom-bretagne.eu>
Date: Fri, 28 Oct 2022 08:37:51 +0200
Subject: [PATCH] Fix LLVM_DYLIB usage

---
 tools/driver/CMakeLists.txt | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/tools/driver/CMakeLists.txt b/tools/driver/CMakeLists.txt
index 1968b97..85a4bf2 100644
--- a/tools/driver/CMakeLists.txt
+++ b/tools/driver/CMakeLists.txt
@@ -9,9 +9,9 @@
 # Keep symbols for JIT resolution
 set(LLVM_NO_DEAD_STRIP 1)
 
+  set(LLVM_LINK_COMPONENTS support)
 if(BUILD_SHARED_LIBS)
   set(LIBS
-    LLVMSupport
 
     clangFrontendTool
 
@@ -25,8 +25,6 @@ if(BUILD_SHARED_LIBS)
   )
 else()
   set(LIBS
-    LLVMSupport
-
     clangASTMatchers
     clangFrontendTool
 
-- 
2.37.3

