From fa3f7a29e1b13fd44c0f4b8606cfc23d89020e30 Mon Sep 17 00:00:00 2001
From: gouarin <loic.gouarin@gmail.com>
Date: Fri, 1 Dec 2017 10:02:31 +0100
Subject: [PATCH] remove jupyter

---
 CMakeLists.txt       | 2 +-
 tools/CMakeLists.txt | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bcc8e4be..179b49cb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -531,7 +531,7 @@ if (CLING_BUILT_STANDALONE OR CMAKE_VERSION VERSION_EQUAL 3 OR
     ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/ClingConfig.cmake
     DESTINATION lib${LLVM_LIBDIR_SUFFIX}/cmake/cling)
 
-  install(DIRECTORY tools/Jupyter/kernel DESTINATION share/cling/Jupyter)
+#  install(DIRECTORY tools/Jupyter/kernel DESTINATION share/cling/Jupyter)
 
 
   # Also copy ClingConfig.cmake to the build directory so that dependent projects
diff --git a/tools/CMakeLists.txt b/tools/CMakeLists.txt
index 831fed03..e16bd462 100644
--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -10,7 +10,7 @@
 if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/../lib/UserInterface/textinput
    OR CLING_INCLUDE_TESTS)
   add_subdirectory(driver)
-  add_subdirectory(Jupyter)
+#  add_subdirectory(Jupyter)
   add_subdirectory(libcling)
   add_subdirectory(demo)
 endif()
-- 
2.14.1

