From 28bcc944af88bd16191cf1c76072c242a181f2a1 Mon Sep 17 00:00:00 2001
From: Sylvain Corlay <sylvain.corlay@gmail.com>
Date: Mon, 30 Apr 2018 01:28:19 +0200
Subject: [PATCH] ast-fix

---
 lib/Utils/AST.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/lib/Utils/AST.cpp b/lib/Utils/AST.cpp
index adb3df3..760dcdf 100644
--- a/lib/Utils/AST.cpp
+++ b/lib/Utils/AST.cpp
@@ -1612,6 +1612,12 @@ namespace utils {
       return QT;
     }
 
+    // Strip deduced types.
+    if (const AutoType* AutoTy = dyn_cast<AutoType>(QT.getTypePtr())) {
+      if (!AutoTy->getDeducedType().isNull())
+        return GetFullyQualifiedType(AutoTy->getDeducedType(), Ctx);
+    }
+
     // Remove the part of the type related to the type being a template
     // parameter (we won't report it as part of the 'type name' and it is
     // actually make the code below to be more complex (to handle those)
-- 
2.5.3

