From c1788189326fbaf678b4a199d56fd0146884c54c Mon Sep 17 00:00:00 2001
From: Sylvain Corlay <sylvain.corlay@gmail.com>
Date: Fri, 6 Dec 2019 17:04:40 +0100
Subject: [PATCH 1/7] Export all symbols on windows

---
 tools/libcling/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tools/libcling/CMakeLists.txt b/tools/libcling/CMakeLists.txt
index 2a2b15e0..beda175f 100644
--- a/tools/libcling/CMakeLists.txt
+++ b/tools/libcling/CMakeLists.txt
@@ -94,6 +94,10 @@ add_cling_library(libcling ${ENABLE_SHARED} ${ENABLE_STATIC}
 set_target_properties(libcling
         PROPERTIES ENABLE_EXPORTS 1)
 
+if(MSVC)
+  set_target_properties(libcling PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS 1)
+endif()
+
 if(ENABLE_SHARED)
   if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     set(LIBCLING_LINK_FLAGS " -Wl,-compatibility_version -Wl,1")
