From 1fdac5afc7a923aa93e3e95b5bf6d83d671a7c99 Mon Sep 17 00:00:00 2001
From: Sylvain Corlay <sylvain.corlay@gmail.com>
Date: Fri, 6 Dec 2019 18:57:22 +0100
Subject: [PATCH 3/7] Drop compile-time sysroot

---
 lib/Interpreter/CIFactory.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/lib/Interpreter/CIFactory.cpp b/lib/Interpreter/CIFactory.cpp
index aabb4d81..9e8758d2 100644
--- a/lib/Interpreter/CIFactory.cpp
+++ b/lib/Interpreter/CIFactory.cpp
@@ -344,10 +344,6 @@ namespace {
           sArguments.addArgument("-nostdinc++");
       }
 
-  #ifdef CLING_OSX_SYSROOT
-    sArguments.addArgument("-isysroot", CLING_OSX_SYSROOT);
-  #endif
-
 #endif // _MSC_VER
 
       if (!opts.ResourceDir && !opts.NoBuiltinInc) {
