From b9e1f9032e5d0f808fa5a067b9a732d9ab1d25ed Mon Sep 17 00:00:00 2001
From: Sylvain Corlay <sylvain.corlay@gmail.com>
Date: Fri, 6 Dec 2019 18:57:22 +0100
Subject: [PATCH 4/8] Drop compile-time sysroot

---
 lib/Interpreter/CIFactory.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/lib/Interpreter/CIFactory.cpp b/lib/Interpreter/CIFactory.cpp
index a546404b..4e8c06cc 100644
--- a/lib/Interpreter/CIFactory.cpp
+++ b/lib/Interpreter/CIFactory.cpp
@@ -341,10 +341,6 @@ namespace {
           sArguments.addArgument("-nostdinc++");
       }
 
-  #ifdef CLING_OSX_SYSROOT
-    sArguments.addArgument("-isysroot", CLING_OSX_SYSROOT);
-  #endif
-
 #endif // _MSC_VER
 
       if (!opts.ResourceDir && !opts.NoBuiltinInc) {
