From b99403a77bcf2829a4a52be2b6ac1597b4b38690 Mon Sep 17 00:00:00 2001
From: Sylvain Corlay <sylvain.corlay@gmail.com>
Date: Fri, 6 Dec 2019 18:57:22 +0100
Subject: [PATCH 4/7] Drop compile-time sysroot

---
 lib/Interpreter/CIFactory.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/lib/Interpreter/CIFactory.cpp b/lib/Interpreter/CIFactory.cpp
index aec15324..f50eff26 100644
--- a/lib/Interpreter/CIFactory.cpp
+++ b/lib/Interpreter/CIFactory.cpp
@@ -344,10 +344,6 @@ namespace {
           sArguments.addArgument("-nostdinc++");
       }
 
-  #ifdef CLING_OSX_SYSROOT
-    sArguments.addArgument("-isysroot", CLING_OSX_SYSROOT);
-  #endif
-
 #endif // _MSC_VER
 
       if (!opts.ResourceDir && !opts.NoBuiltinInc) {
