From 8c197855364dd2d4e0e5d3ef6e9ed8563672c72e Mon Sep 17 00:00:00 2001
From: Sylvain Corlay <sylvain.corlay@gmail.com>
Date: Fri, 28 Aug 2020 22:13:04 +0200
Subject: [PATCH 5/8] enable redefinition allowed

---
 include/cling/Interpreter/RuntimeOptions.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/cling/Interpreter/RuntimeOptions.h b/include/cling/Interpreter/RuntimeOptions.h
index 0240bd14..4dc12cbb 100644
--- a/include/cling/Interpreter/RuntimeOptions.h
+++ b/include/cling/Interpreter/RuntimeOptions.h
@@ -14,7 +14,7 @@ namespace cling {
     /// \brief Interpreter configuration bits that can be changed at run-time
     /// by the user, e.g. to enable/disable extensions.
     struct RuntimeOptions {
-      RuntimeOptions() : AllowRedefinition(0) {}
+      RuntimeOptions() : AllowRedefinition(1) {}
 
       /// \brief Allow the user to redefine entities (requests enabling the
       /// `DefinitionShadower` AST transformer).
