From b2c11036bd46773fd0ec7fdf0cfc6201486e98eb Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Tue, 18 Mar 2025 12:25:13 +1100
Subject: [PATCH 07/10] don't use stale llvmdir

---
 lib/Interpreter/CIFactory.cpp | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/lib/Interpreter/CIFactory.cpp b/lib/Interpreter/CIFactory.cpp
index f50eff26..60c9902c 100644
--- a/lib/Interpreter/CIFactory.cpp
+++ b/lib/Interpreter/CIFactory.cpp
@@ -178,7 +178,6 @@ namespace {
 #endif
 
   static std::string getResourceDir(const char* llvmdir) {
-    if (!llvmdir) {
       // FIXME: The first arg really does need to be argv[0] on FreeBSD.
       //
       // Note: The second arg is not used for Apple, FreeBSD, Linux,
@@ -192,13 +191,6 @@ namespace {
       //
       return CompilerInvocation::GetResourcesPath(
           "cling", (void*)intptr_t(GetExecutablePath));
-    } else {
-      std::string resourcePath;
-      llvm::SmallString<512> tmp(llvmdir);
-      llvm::sys::path::append(tmp, "lib", "clang", CLANG_VERSION_STRING);
-      resourcePath.assign(&tmp[0], tmp.size());
-      return resourcePath;
-    }
   }
 
   ///\brief Adds standard library -I used by whatever compiler is found in PATH.
